modules/diskq/logqueue-disk.h

```diff
@@ -59,6 +59,6 @@ void log_queue_disk_restart_corrupted(LogQueueDisk *self);
 
 
 LogMessage *log_queue_disk_read_message(LogQueueDisk *self, LogPathOptions *path_options);
-gboolean log_queue_disk_write_message(LogQueueDisk *self, LogMessage *msg);
+gboolean log_queue_disk_write_message(LogQueueDisk *self, GString *serialized);
```

Comment:
The symmetry is broken here with this change. The name of the function might have to be adjusted as well.